### PR TITLE
Fe/styles

### PR DESCRIPTION
**Pull request by shehlaMushtaqTkxel**

### CODE REVIEW

File: src/routes/index.tsx [hunk 1]
```diff
@@ -7,7 +7,7 @@ const AppRoutes = () => {
     <Routes>
       <Route path="/sign-in" element={<AdminSignIn />} />
       <Route path="/admin/*" element={<AdminRoutes />} />
-      <Route path="/" element={<Navigate to="/sign-in" />} />
+      <Route path="/" element={<>check</>} />
```
Comment (mamoon-tkxel) [inline src/routes/index.tsx:10]: Revert this

File: src/main.tsx [hunk 2]
```diff
@@ -3,3 +3,13 @@ Promise.all([import("@/Root"), import("@/App")]).then(
...
+import ReactDOM from "react-dom/client";
+import App from "./App.tsx";
+import "./index.css";
+import './styles/base/normalize.scss'
+import './styles/base/_fonts.scss'
+import './styles/base/_typography.scss'
+// import './styles/base/globalStyles.scss'
+import './styles/base/input.scss'
+
+ReactDOM.createRoot(document.getElementById("root")!).render(<App />);
```
Comment (mamoon-tkxel) [inline src/main.tsx:15]: why you have added this line.

Comment (shehlaMushtaqTkxel) [reply src/main.tsx:15]: because i want to access it globally

File: src/routes/index.tsx (hunk 1 above)
Comment (shehlaMushtaqTkxel) [reply src/routes/index.tsx]: check now

**Pull request merged**